### ISSUE TITLE
None of the ports were configured to use SSL certificate

### ISSUE THREAD

**Issue by fubaristic** [dymosoftware/DCD-SDK-Sample#44]
Hi,

When I right click / diagnose the web service in Edge (v97), I get "None of the ports were configured to use SSL certificate."  This also happens when I paste the URL into Chrome.  I looked thru the 4 postings on this page mentioning this error and none of these help or apply.

This is Windows 10, v8.7.4.48936 DLS, v3 DLFW.  The service is running on port 41951 as the framework is found in IE 11, but not in Edge  or Chrome.  I have uninstalled/re-installed more than once, verified the cert is trusted, and rebooted the PC more than once.  We have many installations of this, but this particular one isn't working for us.  

When trace mode is on in the browser development tools, I see that checkLegacyPlugins in IE 11 mentions "ActiveXObject", and in Edge "non-IE".  IE 11 returns isFrameworkInstalled=true, and Edge isFrameworkInstalled=False. I only have the minimized javascript so there is only so far I could go with debugging code in the browser.

Any ideas ?

Thanks !

**Comment by fubaristic** [comment]
I don't think this has anything to do with the javascript API.  This happens with the framework by just right clicking the web service icon / diagnose.   A popup with the "None of the ports were configured to use SSL certificate" is displayed.  There is no browser interaction.

As this is Dymo label framework, I'm guessing the beta-version of dymo connect is not applicable.

**Comment by dymosoftware** [comment]
I provided js file because it is able to easy support for debugging code in the browser.
Yes, this is related with DLS web service tray icon.
You already verified DYMO Root Certificate is trusted after reinstalling DLS which is most web service start issue.
Is your issue "None of the ports were configured to use SSL certificate."?
Have you seen "The service is running on port 41951" because you mentioned on with some browser?

"None of the ports were configured to use SSL certificate." is able to seen if you do not reboot the computer after installing. However you already rebooted.

Are you Admin user with your machine?
If you want to keep DLS app, then how about un-install printers, and remove DLS app, and rebooting, and re-installation, and rebooting?

**Comment by fubaristic** [comment]
I have verified the DYMO root certificate is trusted.

The issue is "None of the ports were configured to use SSL certificate".  This computer has been rebooted many times now, and this message still pops up when the Diagnose option is selected.  Yes, I am an administrative user on the PC.

I can see the service is running on port 41951 on this computer, but the Diagnose option has never worked.  Also, unrelated, my javascript code is never finding the framework.  As mentioned though, the problem is independent of my software.

I'll post back after I try the suggested: un-install printers, and remove DLS app, and rebooting, and re-installation, and rebooting.  I haven't done the un-install printers portion.

Thanks.

**Comment by dymosoftware** [comment]
What is your OS?
Are you going to plan to upgrade to recent release which you can find it from dymo.com?

**Comment by fubaristic** [comment]
Windows 10 is the OS.  As for a recent upgrade, I'm unclear what that is.  DLS is at 8.7.4, which is the latest mentioned at https://www.dymo.com/compatibility-chart.html .  Also, that is the version mentioned when trying to update via the DYMO Label v8 software.
